rr_platform/launch/pointgrey_camera.launch

```diff
@@ -21,30 +22,35 @@
     <param name="format7_color_coding" value="raw8"/>
     <param name="frame_rate" value="60" />
     <param name="auto_exposure" value="false"/>
-    <param name="exposure" value="1.25"/>
+    <param name="exposure" value="0.001"/>
     <param name="auto_brightness" value="false"/>
     <param name="brightness" value="0"/>
-    <param name="auto_white_balance" value="false"/>
+    <param name="auto_white_balance" value="true"/>
     <param name="auto_shutter" value="false"/>
-    <param name="auto_gain" value="false"/>
+    <param name="auto_gain" value="true"/>
```

Comment:
We found that changing these values changed neither the image quality nor the values in rqt_reconfigure. We used reconfigure at competition to adjust the camera params